cibyl/sources/elasticsearch/api.py

```diff
@@ -355,6 +364,125 @@ def get_deployment(self, **kwargs):
 
         return AttributeDictValue("jobs", attr_type=Job, value=job_objects)
 
+    @speed_index({'base': 3})
+    def get_tests(self, **kwargs):
+        """
+            Get tests for a elasticsearch job.
+
+            :returns: container of jobs with the last completed build
+            (if any) and the tests
+            :rtype: :class:`AttributeDictValue`
+        """
+        jobs_found = self.get_jobs(**kwargs)
+        query_body = {
+            "query": {
+                "bool": {
+                    "must": [
+                        {},
+                        {
+                            "bool": {
+                              "should": [
+                                {}
+                              ]
+                            }
+                        },
+                        {
+                            "exists": {
+                                "field": "test_status"
+                            }
+                        },
+                        {
+                            "exists": {
+                                "field": "test_name"
+                            }
+                        }
+                    ]
+                }
+            },
+            "sort": [{"timestamp.keyword": {"order": "desc"}}]
+        }
+
+        build_numbers = []
+        if 'build_number' in kwargs:
+            build_numbers = [status.upper()
+                             for status in
+                             kwargs.get('build_number').value]
+
+        results = []
+        hits = []
```

Comment:
As far as I can tell, 'hits' and 'results' are copies of one another. I think you can get rid of one of them.

Reply:
You're right. Before I needed to access to some index of `results` now it's not necessary. I deleted it.